.github/workflows/validate-alternative.yml

```diff
@@ -0,0 +1,9 @@
+name: Validate JSONs Alternative
```

Comment:
What do you mean by "JSONs Alternative"?

Reply:
Just that it was one of a number of ways of doing the validation. My first was https://github.com/peternewman/rdm-schema/tree/validation which still seems to fail, I think possibly as the validator is older than the schema you're now using.

Given the various online and CLI validators seem to catch different things, running a suite of them probably makes sense, but maybe I should name them based on which tool is doing the validation for each run.